lib/salus/scanners/gosec.rb

```diff
@@ -190,10 +190,15 @@ def config_options
           # exlude the folders from scan
           # can be files or directories
           'exclude-dir': :file
-        }
+        },
+        config_overrides: { 'exclude' => excluded_ids }
       )
     end
 
+    def excluded_ids
+      (fetch_exception_ids + @config.fetch('exclude', [])).uniq
```

Comment:
is there a reason to also `sort`?

Reply:
i don't think so we just pass the exceptions to gosec for exclusion